data/Audiences.ts

```diff
@@ -2,31 +2,28 @@ import { Audience } from "../types/targeting";
 
 const audiences: { [id: string]: Audience } = {
   all_english: {
-    name: "All English users (release)",
+    name: "All English users",
     description: "All users in en-* locales using the release channel.",
-    targeting: "localeLanguageCode == 'en' && browserSettings.update.channel == 'release'",
-    desktop_telemetry:
-      "STARTS_WITH(environment.settings.locale, 'en') AND normalized_channel = 'release'",
-    firefox_channel: "Release",
+    targeting:
+      "localeLanguageCode == 'en' && browserSettings.update.channel == '{firefox_channel}'",
+    desktop_telemetry: "STARTS_WITH(environment.settings.locale, 'en')",
```

Comment:
Q: Does the `desktop_telemetry` also need to specify a `{firefox_channel}`? Previously we were hardcoding `AND normalized_channel = 'release'` but not sure if we need to add that logic back:

```js
"STARTS_WITH(environment.settings.locale, 'en') AND normalized_channel ='{firefox_channel}'"
```

Reply:
In theory we could but this would have to be filled in by jetstream and not experimenter so the formatting will be dependent on @tdsmith 's feedback